mpmath/matrices/matrices.py

```diff
@@ -988,8 +990,8 @@ def mnorm(ctx, A, p=1):
             p = ctx.convert(p)
         m, n = A.rows, A.cols
         if p == 1:
-            return max(ctx.fsum((A[i,j] for i in range(m)), absolute=1) for j in range(n))
+            return max((ctx.fsum((A[i,j] for i in range(m)), absolute=1) for j in range(n)), default=0)
```

Comment:
Why 1?  I would expect 0 here.

Reply:
Was just playing with values and forgot to change it back 👍